lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -412,15 +412,25 @@ protected function transformReportbackItem($data) {
    *   An object containing properties of Signup data:
    *   - id: (string) The Signup id.
    *   - created_at: (string) Date Signup was created.
-   *   - campaign: (string) Campaign Signup belongs to.
    *   - campaign_run: (string) Campaign run Signup belongs to.
+   *   - uri: (string) API URI for Reportback Item data.
+   *   - campaign: (string) Campaign Signup belongs to.
    * @return array
    */
-  protected function transformSignup($data) {
+  protected function transformSignup($data, $current = false) {
     return [
       'id' => $data->id,
       'created_at' => $data->created_at,
+      'campaign_run' => [
+        'id' => $data->campaign_run,
+        'current' => $current,
+        'timing' => [
+          'start' => 'start-date-placeholder',
+          'end' => 'end-date-placeholder',
+        ],
+      ],
       'uri' => $data->uri,
+      'reportbacks' => $data->reportbacks,
```

Comment:
Reportbacks should be included in the `SignupTransformer` after calling `$data['campaign'] = $this->transformCampaign((object) $item->campaign);` and sending to the `$this->transformReportback()` method.